source/locale/pt_BR/LC_MESSAGES/using-api-interface.po

```diff
@@ -404,42 +403,41 @@ msgstr "Conferindo dados históricos"
 
 #: ../../source/using-api-interface.rst:376
 msgid ""
-"In order to check all values that were sent from a device for a "
-"particular attribute, you could use the history api, see more in :doc"
-":`components-and-apis`. Let's first send a few other values to dojot so "
-"we can get a few more interesting results:"
+"In order to check all values that were published from a device for a particular "
+"attribute, you could use the history api, see more in :doc:`components-and-"
+"apis`. Let's first send a few other values to dojot so we can get a few more "
+"interesting results:"
 msgstr ""
-"A fim de se conferir todos os valores que foram enviados pelo dispositivo"
-" para um atributo particular, pode-se utilizar a API do History, veja "
-"mais em :doc:`components-and-apis`. Vamos, então, enviar agora alguns "
-"outros valores à dojot para que possamos conseguir resultados um pouco "
-"mais interessantes:"
+"A fim de se conferir todos os valores que foram publicados pelo dispositivo "
+"para um atributo particular, pode-se utilizar a API do History, veja mais em :"
+"doc:`components-and-apis`. Vamos, então, enviar agora alguns outros valores à "
+"dojot para que possamos conseguir resultados um pouco mais interessantes:"
 
 #: ../../source/using-api-interface.rst:387
-msgid "To retrieve all values sent for temperature attribute of this device:"
+msgid "To retrieve all values published for temperature attribute of this device:"
 msgstr ""
-"Para recuperar todos os valores enviados do atributo ``temperature`` "
-"desse dispositivo:"
+"Para recuperar todos os valores publicados do atributo ``temperature`` desse "
```

Comment:
enviados

Reply:
Changed for "Para recuperar todos os valores enviados dos atributos"